OracleDatabase/OracleRealApplicationClusters/README.md

```diff
@@ -0,0 +1,474 @@
+
+- Allocate block devices of at least 50GB for OCR/Voting and databases files. You can refer Oracle 12.2 GRID and database install guides for details on recommended number of devices and total storage size. 
+
+- You must configure the following addresses manually in your DNS:
+
+	- Public IP address for each container
+
+	- Private IP address for each container
+
+	- Virtual IP address for each container
+
+	- Three single client access name (SCAN) addresses for the cluster.
+
+	For details, please refer to following guide:
+	https://docs.oracle.com/database/122/LADBI/LADBI.pdf
+
+- You need to plan your private and public network for containers before you start installation. You can create a network bridge on every host so containers running within that host can communicate with each other.  For example, create rac_pub1_nw for the public network (172.15.1.0/24) and rac_priv1_nw (192.168.17.0/24) for a private network. You can use any network subnet for testing. However, in this document we reference the public network on 172.15.1.0/24 and the private network on 192.168.17.0/24.
+
+		#docker network create --driver=bridge --subnet=172.15.1.0/24 rac_pub1_nw
+
+		#docker network create --driver=bridge --subnet=192.168.17.0/24 rac_priv1_nw
+
+		Note: You must run  RAC on Docker on multi-host using MACVLAN Docker driver (see Docker documentation).
+   
+- If the docker bridge network is not available outside your host, you can use the Oracle Connection Manager (CMAN) image to access the RAC Database from outside the host. You need to download and setup Oracle Connection Manager image. For the setup, please refer README of Oracle Connection Manager image. 
+
+- RAC needs to run certain processes in real time mode. To run processes inside a container in real time mode, you need to make changes in your docker configuration files. Please update OPTIONS value in /etc/sysconfig/docker to following:
+
+	OPTIONS='--selinux-enabled --cpu-rt-runtime=950000 --cpu-rt-period=1000000         --exec-opt native.cgroupdriver=systemd'
+
+	Once you have editied the /etc/sysconfig/docker, execute following commands:
+
+		#systemctl daemon-reload
+
+		#systemctl stop docker
+
+		#systemctl start docker
+
+	For details, please refer
+    	https://docs.docker.com/engine/reference/commandline/dockerd/#examples
```

Comment:
Please link to the Oracle documentation:  https://docs.oracle.com/cd/E52668_01/E87205/html/section_yqk_dqg_gp.html